src/components/backpack/backpack.jsx

```diff
@@ -62,18 +66,24 @@ const Backpack = ({contents, error, expanded, loading, onToggle}) => (
                         </div>
                     ) : (
                         contents.length > 0 ? (
-                            <div className={styles.backpackListInner}>
+                            <div
+                                className={classNames(styles.backpackListInner, {
+                                    [styles.dragOver]: dragOver
+                                })}
+                            >
                                 {contents.map(item => (
                                     <SpriteSelectorItem
                                         className={styles.backpackItem}
                                         costumeURL={item.thumbnailUrl}
                                         details={item.name}
                                         dragPayload={item}
                                         dragType={dragTypeMap[item.type]}
+                                        id={item.id}
```

Comment:
I had to do a double-take on this prop name. I checked to remember how it's used and it doesn't cause an issue (we never pass `id` down to the rendered component). But we should consider using a different prop name than `id` for this since it's easily read as the HTML `id` attribute, which would be inappropriate to use in a reusable component.

Reply:
(I didn't mean to change this now, just a sidebar note)